tools/tf2openapi/README.md

```diff
@@ -0,0 +1,21 @@
+# Tf2OpenAPI
+The main purpose of this tool is to enable automatically generating OpenAPI 3.0 specifications for prediction HTTP requests from TensorFlow SavedModel files containing signature definitions (SignatureDefs) for models. One use case for the OpenAPI specification involves generating sample payloads in a UI.
+
+## Overview
+###Use Cases for OpenAPI
```

Comment:
This formatting is bad. Try testing these out before posting. I use the vscode markdown tool

Reply:
Thanks for catching that. I had previewed this in IntelliJ, and it had showed up fine...